pages/emailSender.tsx

```diff
@@ -118,6 +122,16 @@ const EmailSender: NextPage = () => {
     return headersArrFinal
   }
 
+  const sendEmails = () => {
+    const localErrorMessages : ErrorMessage[] = []
+    for (let i = 0; i < finalMessages.length; i++) {
+      if (i % 2 === 0) {
+        localErrorMessages.push({ id: finalMessages[i].id, message: 'Errorrrr!' })
+      }
+    }
+    setResultErrorMessage({ errorMessages: localErrorMessages, resultMessage: { isError: localErrorMessages.length > 0, message: localErrorMessages.length > 0 ? `Error sending ${localErrorMessages.length} of ${finalMessages.length}` : 'Sent emails successfully' } })
```

Comment:
Nit: hmm this line's a little long - does the eslint hook check for that?

Reply:
No I don't think it does - just added!